docs/devel/cherry-picks.md

```diff
@@ -42,12 +42,13 @@ depending on the point in the release cycle.
 
 1. Cherrypicks are [managed with labels and milestones]
 (pull-requests.md#release-notes)
-1. All label/milestone accounting happens on PRs on master. There's nothing to
-do on PRs targeted to the release branches.
 1. When you want a PR to be merged to the release branch, make the following
 label changes to the **master** branch PR:
-  * Remove release-note-label-needed
-  * Add an appropriate release-note-(!label-needed) label
+  * Remove the `release-note-label-needed` label
```

Comment:
This line can be removed, since it applies to all PRs against master and you don't have to do anything special for a cherry pick candidate.